static/app/components/globalSelectionLink.tsx

```diff
@@ -7,6 +6,7 @@ import Link from 'sentry/components/links/link';
 import {extractSelectionParameters} from 'sentry/components/organizations/globalSelectionHeader/utils';
 
 type Props = WithRouterProps & {
+  children?: React.ReactChildren;
```

Comment:
should we replace `React.ReactChildren` with `React.ReactNode` here?

Reply:
Yeah.. it's not totally clear to me when to use which haha